lib/utils/buildDocs.js

```diff
@@ -32,14 +32,26 @@ const collectSchemas = InitSchema => {
   return schemas;
 };
 
-const quoteOrNa = val => (val ? `\`${val.replace('`', '')}\`` : '_n/a_');
+const BREAK_LENGTH = 96;
+const prepQuote = val => val.replace('`', '');
+const quote = (val, triple, prepend = '') =>
+  triple && val.length > BREAK_LENGTH
+    ? `\`\`\`\n${val
+        .match(/[^\r\n]+/g)
```

Comment:
this section is a little opaque. could you add comments about the desired before and after?

Reply:
👍